examples/dicklang/parse_test.go

```diff
@@ -0,0 +1,38 @@
+// This is an example for the github.com/romshark/llparser library
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParser(t *testing.T) {
+
```

Comment:
This is just a small cosmetic notice but I never add spaces in the beginning and end of a block.

Reply:
When I remove the spaces, `gofmt` gives me trouble